DfE.FindInformationAcademiesTrusts/Pages/OtherServicesLinkBuilder.cs

```diff
@@ -1,14 +1,17 @@
-using DfE.FindInformationAcademiesTrusts.Services.Trust;
+using DfE.FindInformationAcademiesTrusts.Data.Enums;
 
 namespace DfE.FindInformationAcademiesTrusts.Pages;
 
 public interface IOtherServicesLinkBuilder
 {
     string GetInformationAboutSchoolsListingLinkForTrust(string trustUid);
     string GetInformationAboutSchoolsListingLinkForAcademy(string urn);
-    string? CompaniesHouseListingLink(TrustDetailsServiceModel trust);
-    string? SchoolFinancialBenchmarkingServiceListingLink(TrustDetailsServiceModel trust);
-    string? FindSchoolPerformanceDataListingLink(TrustDetailsServiceModel trust);
+    string? CompaniesHouseListingLink(string? companiesHouseNumber);
+
+    string? SchoolFinancialBenchmarkingServiceListingLink(TrustType trustType, string? satAcademyUrn,
+        string? companiesHouseNumber);
+
+    string? FindSchoolPerformanceDataListingLink(string uid, TrustType trustType, string? satAcademyUrn);
```

Comment:
Can you remove the inconsistent whitespace please?
```suggestion
    string? CompaniesHouseListingLink(string? companiesHouseNumber);
    string? SchoolFinancialBenchmarkingServiceListingLink(TrustType trustType, string? satAcademyUrn, string? companiesHouseNumber);
    string? FindSchoolPerformanceDataListingLink(string uid, TrustType trustType, string? satAcademyUrn);
```

Reply:
Ah, this bothered me too - Rider was pretty insistent that I have the space but I can try to reconfigure it!

Reply:
Have rearranged to keep extra spaces at the end